.github/workflows/ci.yml

```
@@ -22,6 +22,11 @@ jobs:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
+        
+    - if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/heads/main') }}
+      uses: akhilmhdh/contributors-readme-action@v2.3.10
+      env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)